packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -169,34 +169,34 @@ const iconIsButtonSelector = `&:has(button.${iconClassName})`;
 export const inputWrapperSizeStyles: Record<Size, string> = {
   [Size.XSmall]: css`
     height: 22px;
-    padding-inline: 6px;
+    padding-inline: ${spacing[200]}px;
 
     ${iconIsButtonSelector} {
-      padding-inline-end: 4px;
+      padding-inline-end: ${spacing[100]}px;
     }
   `,
   [Size.Small]: css`
     height: 28px;
-    padding-inline: 6px;
+    padding-inline: ${spacing[200]}px;
 
     ${iconIsButtonSelector} {
-      padding-inline-end: 4px;
+      padding-inline-end: ${spacing[100]}px;
     }
   `,
   [Size.Default]: css`
     height: 36px;
-    padding-inline: 12px;
+    padding-inline: ${spacing[300]}px;
 
     ${iconIsButtonSelector} {
-      padding-inline-end: 6px;
+      padding-inline-end: ${spacing[150]}px;
     }
   `,
   [Size.Large]: css`
     height: 48px;
-    padding-inline: 16px;
+    padding-inline: ${spacing[300]}px;
```

Comment:
are we updating the spacing intentionally? if not should be 400

Reply:
yes, this is an intentional update according to the spacing audit here: https://www.figma.com/file/WmbvStSle8GtLg5RqEt2oh/Component-Consistency-Project?type=design&node-id=45%3A1052&mode=dev&t=x29rW9Kcf3OpO8GP-1